packages/core-contracts/contracts/proxy/OwnableUUPSImplementation.sol

```diff
@@ -0,0 +1,68 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./UniversalProxyImplementation.sol";
+import "../ownership/Ownable.sol";
+
+contract OwnableUUPSImplementation is UniversalProxyImplementation, Ownable {
+    struct UnstructuredProxyStorage {
```

Comment:
You only need to destructure  a single variable in storage, the implementation address. That's it, no storage namespaces at all.